packages/flutter_tools/bin/podhelper.rb

```diff
@@ -27,8 +27,17 @@ def flutter_ios_podfile_setup
 # end
 # @param [PBXAggregateTarget] target Pod target.
 def flutter_additional_ios_build_settings(target)
+  return unless target.platform_name == :ios
+
+  # [target.deployment_target] is a [String] formatted as "8.0".
+  inherit_deployment_target = target.deployment_target[/\d+/].to_i < 9
```

Comment:
should we leave a breadcrumb comment at the location in the tool where we define the min version to also update this magic constant?

Reply:
Good idea.